README.md

```diff
@@ -11,6 +11,9 @@ See the [player's manual](https://github.com/endless-sky/endless-sky/wiki/Player
 # Installing the game
 Official releases of Endless Sky are available as direct downloads from [GitHub](https://github.com/endless-sky/endless-sky/releases) and on [Steam](https://store.steampowered.com/app/404410/Endless_Sky/). A PPA is available for [Ubuntu](https://launchpad.net/~mzahniser/+archive/ubuntu/endless-sky). Other package managers may also include the game, though the specific version provided may not be up-to-date.
 
+### Mac direct download info:
+The .dmg that is provided by GitHub is an unsigned application. This means you have to tell your Mac that this file is safe to run. On your first try to run the app a message will pop up that this app is not from a verified developer (important: you have to try running it at least once). To run it anyways go to Settings->Privacy & Security->Security. In a small box you will see a message that this game is not from an indentified developer. Right below that message there is a button that says "Open Anyway". Click it and macOS will ask you for your password, enter your password and the app will finally launch.
```

Comment:
```suggestion
### Mac direct download info
The .dmg that is provided by GitHub is an unsigned application. This means you have to tell your Mac that this file is safe to run. On your first try to run the app, press the Control (CTRL) key and click on the application, then click "Open." In a small box you will see a message that this game is not from an identified developer. In that box, click the button that says "Open." macOS will then ask you for your password. Enter your password and the app will launch.
```

Reply:
"Control (CTRL)," ".dmg," and "Open" can also all have code blocks, with the quotes around "Open" removed. This takes into account the other suggestion by tibetiroka.